src/patternfly/components/Button/button.scss

```diff
@@ -125,12 +125,13 @@
   border-radius: var(--pf-c-button--BorderRadius);
 
   .pf-c-button__icon {
```

Comment:
can you move this selector block outside of `.pf-c-button`? doesn't seem like it needs to be nested